packages/components/src/Menu/Menu.css

```diff
@@ -10,28 +10,18 @@
 .menu {
   position: fixed;
   right: 0;
+  bottom: 0;
+  left: 0;
   z-index: var(--elevation-menu);
   width: 100%;
+  max-height: 72vh;
   padding: var(--space-small);
+  padding-bottom: env(safe-area-inset-bottom);
   border: var(--border-base) solid var(--color-grey--lighter);
+  border-radius: var(--radius-base) var(--radius-base) 0 0;
+  overflow-y: scroll;
   background-color: var(--color-white);
-}
-
-@media only screen and (max-width: 639px), print {
-  .menu {
-    bottom: 0;
-    left: 0;
-    max-height: 72vh;
-    padding-bottom: env(safe-area-inset-bottom);
-    border-radius: var(--radius-base) var(--radius-base) 0 0;
-    overflow-y: scroll;
-    transition: transform 300ms ease;
-    -webkit-overflow-scrolling: touch;
-  }
-
-  .menu:last-child {
```

Comment:
We still need this to give the last menu item proper spacing:
![image](https://user-images.githubusercontent.com/39704901/85061974-f2687780-b164-11ea-964a-3ddac62a077b.png)

Reply:
(I'll push this up)